src/services/ReportService.ts

```diff
@@ -356,6 +356,7 @@ const seedbankFromFacility = (facility: Facility, originalReport: Report): Repor
     operationStartedDateEditable: true,
     totalSeedsStored: 0,
     workers: {},
+    selected: true,
```

Comment:
I added this here to compile, but this function is removed in charlie's PR (https://github.com/terraware/terraware-web/pull/1267)